learn/advanced/working_with_dates.md

```diff
@@ -0,0 +1,119 @@
+# Working with dates
+
+In this guide, you will learn about Meilisearch's approach to date and time values, how to prepare your dataset for indexing, and how to chronologically sort and filter search results.
+
+## Preparing your documents
+
+To filter and sort search results chronologically, your documents must have at least one numeric field containing a [UNIX timestamp](https://kb.narrative.io/what-is-unix-time).
+
+As an example, consider a database of video games. In this dataset, the release year is formatted as a timestamp:
+
+```json
+[
+  {
+    "id": 0,
+    "title": "Return of the Obra Dinn",
+    "genre": "adventure",
+    "release_timestamp": 1538949600
+  },
+  {
+    "id": 1,
+    "title": "The Excavation of Hob's Barrow",
+    "genre": "adventure",
+    "release_timestamp": 1664316000
+  },
+  {
+    "id": 2,
+    "title": "Bayonetta 2",
+    "genre": "action",
+    "release_timestamp": 1411164000
+  }
+]
+```
+
+If your date field is expressed in a format other than a numeric timestamp, like [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html), you must convert it before indexing it with Meilisearch.
+
+Most programming languages have built-in tools to help you with this process. The JavaScript example below converts a game's release date, `"2018-10-18"`, to a numeric timestamp:
+
+```js
+let game = {
+  "id": 0,
+  "title": "Return of the Obra Dinn",
+  "genre": "adventure",
+  "release_date": "2018-10-18T00:00Z"
+};
+
+const timestampInMilliseconds = Date.parse(game.year); // Date.parse returns the timestamp in milliseconds
+const timestamp = timestampInMilliseconds / 1000; // UNIX timestamps must be in seconds
+
+game = {
+  "id": 0,
+  "title": "Return of the Obra Dinn",
+  "genre": "adventure",
+  "release_date": "2018-10-18T00:00Z",
+  "release_timestamp": timestamp
+};
+```
+
+:::tip
+When preparing your dataset, it can be useful to leave the original date and time fields in your documents intact. In the example above, we keep the `release_date` field because it is more readable than the raw `timestamp`.
+:::
+
+After adding a numeric timestamp to all documents, [index your data](/reference/api/documents.md#add-or-replace-documents) as usual. See below for an example using our nonexistent video games dataset.
```

Comment:
Technically, we do have a dataset with 3 games, so idk if we should say "nonexistent". We have a similar dataset for the geosearch guide as well

Reply:
If we refer to "our video games dataset" but don't provide a downloadable `videogamesdataset.json` file anywhere on the page, I can only see that confusing and misleading users. I think either we keep it how it is (rewording possible), or we provide an actual video games dataset.

Personally I support the former option, because it's less work and I don't think that a downloadable dataset would add much value to this guide. But maybe I'm wrong?

Reply:
Perhaps `hypothetical` is a better word?

Reply:
How about the wording we used in geosearch:
>Using our [example dataset](https://docs.meilisearch.com/restaurants.json),

Reply:
Followed @maryamsulemani97's suggestion, re-requesting reviews